src/main/java/fi/aalto/cs/apluscourses/model/Main.java

```diff
@@ -0,0 +1,39 @@
+package fi.aalto.cs.apluscourses.model;
+
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class Main {
```

Comment:
could there be maybe a JavaDoc or something that briefly explains the idea of the `Main` class here (not to mention the ambiguity of the Java `Main` class in general) :question: 

another question is related to the logical location of the class, as all the other classes in that package are rather very concrete and represent an A+ entity in a way :question:

Reply:
I thought it would be funny to have model correspondent for `MainViewModel` but now I see it was unnecessary. I'll remove this class.